.gitattributes

```diff
@@ -0,0 +1,7 @@
+/tests                 export-ignore
+/.gitattributes        export-ignore
+/.gitignore            export-ignore
+/.scrutinizer.yml      export-ignore
+/.travis.yml           export-ignore
+/CONTRIBUTING.md       export-ignore
+/phpunit.xml           export-ignore

```

Comment:
No need for leading `/`, attributes are always relative to the directory it is in.